vars/pmebuild.groovy

```diff
@@ -226,4 +227,14 @@ def getDefaultBranch(Map<String, Object> projectConfig, String currentBranch) {
     return projectConfig != null && projectConfig['scmRevision'] ? projectConfig['scmRevision'] : currentBranch
 }
 
+/**
+ * Saves build project ok in ALREADY_BUILT_PROJECTS env var
+ *
+ * @param project the project name (this should match with the builds.project from the file)
+ */
+def saveBuildProjectOk(String project){
+    env.ALREADY_BUILT_PROJECTS = env.ALREADY_BUILT_PROJECTS != null ?: ""
```

Comment:
I think you don't need here the `! =null`

Reply:
improved. Thanks @radtriste